tests/test_utils.go

```diff
@@ -88,6 +88,7 @@ func setupDB(t *testing.T, db *sql.DB) {
 			cid BYTEA NOT NULL,
 			relation TEXT NOT NULL,
 			activated TIMESTAMP,
+			timestamp BIGINT,
```

Comment:
another q: why are we using `BIGINT` instead of `TIMESTAMP`?

Reply:
actually, there's a not very strong reason, besides it felt easier to work with `BIGINT`